.github/workflows/ferretdb_packages.yml

```diff
@@ -81,3 +81,90 @@ jobs:
         run: |
           git status
           git diff --exit-code
+
+  docker:
+    name: Build docker (Pg ${{ matrix.pg }})
+    runs-on: ubuntu-24.04
+    timeout-minutes: 40
+
+    if: >
+      github.event_name != 'pull_request' ||
+      (
+        !contains(github.event.pull_request.labels.*.name, 'not ready') &&
+        contains(github.event.pull_request.labels.*.name, 'packages')
+      )
+
+    strategy:
+      fail-fast: false
+      matrix:
+        pg: [15, 16, 17]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: FerretDB/github-actions/setup-go@main
+
+      - name: Define Docker tags
+        id: tag
+        run: |
+          cd ferretdb_packaging
+          go mod tidy
+          go mod verify
+          go run ./defineversion --command docker-tags
+        env:
+          INPUT_PG_VERSION: ${{ matrix.pg }}
+
+      - name: Debian package version
+        id: version
+        run: |
+          cd ferretdb_packaging
+          go mod tidy
+          go mod verify
+          go run ./defineversion --command deb-version --control-file ../pg_documentdb_core/documentdb_core.control
+
+      - name: Build Debian package ${{ steps.version.outputs.version }} to use for docker build
+        if: steps.version.outputs.version != ''
+        run: ./ferretdb_packaging/build_packages.sh --os deb12 --pg ${{ matrix.pg }} --version ${{ steps.version.outputs.version }}
+
+      - name: Initialize Docker Buildx builder
+        run: make -C ferretdb_packaging docker-init
+
+      # mostly to check that this command still works
+      - name: Build local Docker image
+        if: steps.version.outputs.version != ''
+        run: make -C ferretdb_packaging docker-documentdb FILE=development POSTGRES_VERSION=${{ matrix.pg }} DOCUMENTDB_VERSION=${{ steps.version.outputs.version }}
+
+      - name: Login to Docker Hub
```

Comment:
This step currently fails
```
Run docker/login-action@v3
  with:
    username: ferretdbbot
    ecr: auto
    logout: true
  env:
    GOPATH: /home/runner/go
    GOCACHE: /home/runner/go/cache
    GOLANGCI_LINT_CACHE: /home/runner/go/cache/lint
    GOMODCACHE: /home/runner/go/mod
    GOPROXY: https://proxy.golang.org/
    GOTOOLCHAIN: local
Error: Password required
```

Reply:
Please comment out it and quay.io there and in `defineversion`. We will re-enable them in separate PRs